agent/charms/testflinger-agent-host-charm/src/charm.py

```diff
@@ -79,6 +93,39 @@ def update_testflinger_repo(self):
         self.unit.status = MaintenanceStatus("Cloning testflinger repo")
         testflinger_source.clone_repo(LOCAL_TESTFLINGER_PATH)
 
+    def update_config_files(self):
+        """
+        Clone the config files from the repo and swap it in for whatever is
+        in AGENT_CONFIGS_PATH
+        """
+        config_repo = self.config.get("config-repo")
+        config_dir = self.config.get("config-dir")
```

Comment:
`config_dir` is not used in Repo.clone_from. Is it missed?

Reply:
Correct. That's something we're going to use later, but it was helpful to go ahead and include the config option for it here. I'm happy to consider a different name if you have suggestions for something you think might be more clear. The purpose of `config-dir` is the location inside the repo where you will find the configs for agents this host needs to care about. For example, we might point it at a repo that has configs for multiple labs, but each needs a separate agent-host deployment. So with `config-dir`, you can tell it to only look at configs in the path `ce-cert/data-tel-l3` for example.

An upcoming change will then take all those configs, and generate service files to run the agent for each one of them. But this way it won't look at the configs in other parts of the repo that might be out-of-scope for this lab.

Reply:
Thanks for the clarification. If it is something that we will utilize in upcoming changes then it looks good to keep it as now. The name is very clear for me as my understanding was the same as your definition.  :)